### ISSUE TITLE
Angular 2

### ISSUE THREAD

**Issue by carlos-cubas** [darosh/angular-swagger-ui-material#19]
Any plans to port to ng2?

**Comment by ghost** [comment]
Is it going to happen? Or are there ng v4 alternatives?

**Comment by darosh** [comment]
@poulad-rbc no, this is abandoned project, I work on https://github.com/darosh/oax instead. 

Faster, smarter, better...  :) But unfinished yet.

And without Angular :)

**Comment by ghost** [comment]
@darosh Seems interesting. But any specific reason for moving away from Angular to Vue?

**Comment by darosh** [comment]
@pouland-rbc it is just another app, ...using different approach ...and framework
